lib/webpacker/file_loader.rb

```diff
@@ -7,9 +7,17 @@ class FileLoaderError < StandardError; end
   attr_accessor :data
 
   class << self
+    def exist?
+      instance.data.present? rescue false
+    end
+
     def load(path = file_path)
       self.instance = new(path)
     end
+
+    def reloadable?
+      ["development", "test"].include?(Rails.env)
```

Comment:
Yepp thought about that too but I only wanted to limit this feature explicitly for - test and development. For ex - folks also might have staging and other environments where they don't want to reload. What do you think?